web/server/codechecker_server/api/report_server.py

```diff
@@ -373,21 +376,36 @@ def filter_open_reports_in_tags(results, run_ids, tag_ids):
     return results
 
 
-def get_source_component_file_query(session, component_name):
+def get_include_skip_queries(
+    skip: List[str],
+    include: List[str]
```

Comment:
```suggestion
    include: List[str],
    skip: List[str]
```
It's a minor thing, but couldn't we swap the parameters so their order is the same as the returning tuple?